spec/models/family_spec.rb

```diff
@@ -1,8 +1,18 @@
 require "rails_helper"
 
 describe Family do
-  describe "validations" do
+  context "validations" do
     it { is_expected.to validate_presence_of(:last_name) }
     it { is_expected.to validate_presence_of(:capability) }
   end
+
+  context "by_last_name" do
+    it "orders by last name" do
+      family1 = create(:family, last_name: "Zoo")
+      family2 = create(:family, last_name: "Alba")
+      family3 = create(:family, last_name: "Rossi")
```

Comment:
Shall we change one of these to be lowercase?

Reply:
Done in sha `4f9103b`